README.md

```diff
@@ -4,13 +4,32 @@ This repo contains the code for building http://source.dot.net
 ## Build Status
 [![Build Status](https://dev.azure.com/dnceng/internal/_apis/build/status/dotnet-source-indexer/dotnet-source-indexer%20CI?branchName=master)](https://dev.azure.com/dnceng/internal/_build/latest?definitionId=612&branchName=master)
 
-## Building
+## What Is It?
+This repo uses https://github.com/KirillOsenkov/SourceBrowser (with a few additions here https://github.com/dotnet/SourceBrowser/tree/source-indexer) to index the dotnet sources and produce a navigatable and searchable website containing the full source code. This includes code from the runtime, winforms, wpf, aspnetcore, and msbuild, among others. For a full list see here https://github.com/dotnet/source-indexer/blob/master/src/index/repositories.props.
 
-To clone and build locally (Windows only):
-1. `git clone --recursive https://github.com/dotnet/source-indexer.git`
-2. `cd source-indexer\src\SourceBrowser`
-3. `git fetch --all`
-4. `git checkout source-indexer`
-5. `dotnet restore`
-6. `cd ..\..`
-7. `msbuild build.proj`
+## Build Prerequsites
+The build requires .net core 3.1.201 and Visual Studio 2019 to build.
+
+## Build
+The build will only work on windows because the source indexer executable is a .net framework executable.
+1. `git clone https://github.com/dotnet/source-indexer.git`
```

Comment:
Nit, but I would very much prefer list be numbered explicitly rather than depending on the renderer. Markdown is meant to be easy to read as plain text.